addons/l10n_nl/data/template/account.account-nl.csv

```diff
@@ -1,347 +1,347 @@
-"8041","Turnover NL third-party work","8041","income","l10n_nl.account_tag_11","False","Omzet NL werk derden"
-"8042","Turnover within EU third-party work","8042","income","l10n_nl.account_tag_11","False","Omzet binnen EU werk derden"
-"8043","Turnover outside EU third-party work","8043","income","l10n_nl.account_tag_11","False","Omzet buiten EU werk derden"
-"8044","Turnover Intercompany work third parties","8044","income","l10n_nl.account_tag_11","False","Omzet Intercompany werk derden"
-"8065","Payment discount debtors","8065","income_other","l10n_nl.account_tag_11","False","Betalingskorting debiteuren"
-"8090","Sundry other income","8090","income","l10n_nl.account_tag_11","False","Diverse overige opbrengsten"
-"8110","Installation/remote sales within EU trade goods 1","80011","income","l10n_nl.account_tag_11","False","Installatie/afstandsverkopen binnen EU handelsgoederen 1"
-"8120","Installation/remote sales within EU trade goods 2","80012","income","l10n_nl.account_tag_11","False","Installatie/afstandsverkopen binnen EU handelsgoederen 2"
-"8130","Installation/remote sales within EU trade goods 3","80013","income","l10n_nl.account_tag_11","False","Installatie/afstandsverkopen binnen EU handelsgoederen 3"
-"8920","Exchange rate differences","8920","income","l10n_nl.account_tag_11","False","Koersverschillen"
-"9011","Result of participating interests 1","9011","income_other","l10n_nl.account_tag_16","False","Resultaat deelneming 1"
-"9012","Result of participating interests 2","9012","income_other","l10n_nl.account_tag_16","False","Resultaat deelneming 2"
-"9013","Result of participating interests 3","9013","income_other","l10n_nl.account_tag_16","False","Resultaat deelneming 3"
-"9150","Reorganisation costs","9150","expense","l10n_nl.account_tag_8","False","Reorganisatiekosten"
-"9200","Error account","9200","expense","l10n_nl.account_tag_29","False","Foutenrekening"
-"9900","Corporate tax","9900","expense","l10n_nl.account_tag_33","False","Vennootschapsbelasting"
-"9950","Corporation tax previous years","9950","expense","l10n_nl.account_tag_33","False","Vennootschapsbelasting voorgaande jaren"
+"id","name","code","account_type","tag_ids","reconcile","name@nl","name@de"
+"0010",Acquisition value Goodwill,"0010","asset_fixed","l10n_nl.account_tag_17","False","Aanschafwaarde Goodwill","Anschaffungswert Geschäftswert"
+"0019","Amortisation Goodwill","0019","asset_fixed","l10n_nl.account_tag_17","False","Afschrijving Goodwill","Amortisation Geschäftswert"
+"0020","Acquisition value other intangible assets","0020","asset_fixed","l10n_nl.account_tag_17","False","Aanschafwaarde overige immateriele vaste activa","Anschaffungswert sonstige immaterielle Vermögenswerte"
+"0029","Amortisation of other intangible assets","0029","asset_fixed","l10n_nl.account_tag_17","False","Afschrijving overige immateriele vaste activa","Abschreibungen auf sonstige immaterielle Vermögenswerte"
```

Comment:
In one case Amortisation is translated to Amortisation and in another case to Abschreibungen. Is there a reason why ? If not I guess it should be better to be consistent across the translations :)

I see that Depreciation is also translated to Abschreibungen, so perhaps best to keep Amortisation to differ them ?